### PR TITLE
Add Hebrew translation

### LINKED ISSUE #1689: Help improve Appwrite's Hebrew language translation ✍️
# Introduction

Did you know Appwrite's Locale API has support for returning responses in your own locale based on the value of the `X-Appwrite-Locale` header? 😮 We already support 56 languages and need your help to improve them! 💪

Your task is to update Appwrite's **Hebrew** language translations. We recently added new features, so some translations might be missing. You can find a complete in-depth tutorial in our official [tutorial for adding translations](https://github.com/appwrite/appwrite/blob/master/docs/tutorials/add-translations.md).

This is what missing translation would look like:

```json
"emails.magicSession.footer": "",
```

### Tasks summary:

- [ ] Fork & clone Appwrite
- [ ] Update translations for Hebrew in `app/config/locale/translations/he.json`
- [ ] Test the translations
- [ ] Push changes and submit pull request

If you need any help, reach out to us on our [Discord server](https://discord.gg/GSeTUeA).

Are you ready to work on this issue? 🤔 Let us know, and we will assign it to you 😊

**Happy Appwriting!**

---

### PR DESCRIPTION
**Pull request by Kokoden**
## What does this PR do?

Provides missing Hebrew translations.  I changed the inspiration quote to "To be wise is to know when to overlook" since it sounds more natural in Hebrew, let me know if you want me to change that :)

## Related PRs and Issues

Resolves #1689

### CODE REVIEW

File: app/config/locale/translations/he.json [hunk 1]
```diff
@@ -1,32 +1,32 @@
 {
-    "settings.inspire": "\"The art of being wise is the art of knowing what to overlook.\"",
+    "settings.inspire": "\"להיות חכם זה לדעת מתי להתעלם.\"",
```
Comment (eldadfux) [inline app/config/locale/translations/he.json:2]:
```suggestion
    "settings.inspire": "\"להיות חכם זה לדעת ממה להתעלם.\"",
```

File: app/config/locale/translations/he.json [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+    "emails.magicSession.signature": "צוות {{project}}",
+    "emails.recovery.subject": "איפוס סיסמא",
+    "emails.recovery.hello": "שלום {{name}}",
+    "emails.recovery.body": "עקוב אחר קישור זה כדי לאפס את סיסמת הפרויקט {{project}}.",
+    "emails.recovery.footer": "אם לא ביקשת לאפס את הסיסמה, תוכל להתעלם מהודעה זו.",
+    "emails.recovery.thanks": "תודה",
+    "emails.recovery.signature": "צוות {{project}}",
+    "emails.invitation.subject": "הזמנה לצוות %s ב- %s",
+    "emails.invitation.hello": "שלום",
+    "emails.invitation.body": "דואר זה נשלח אליך מכיוון ש {{owner}} רצה להזמין אותך להיות חבר בצוות {{team}} ב- {{project}}.",
```
Comment (eldadfux) [inline app/config/locale/translations/he.json:26]:
```suggestion
    "emails.invitation.body": "דואר זה נשלח אליך מכיוון ש {{owner}} רצה להזמין אותך להיות חבר בצוות {{team}} ב-{{project}}.",
```

File: app/config/locale/translations/he.json [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
+    "emails.verification.signature": "צוות {{project}}",
+    "emails.magicSession.subject": "כניסה למערכת",
+    "emails.magicSession.hello": "שלום,",
+    "emails.magicSession.body": "לחץ על קישור זה כדי להיכנס.",
+    "emails.magicSession.footer": "אם לא ביקשת להיכנס באמצעות דוא\"ל זה, תוכל להתעלם מהודעה זו.",
+    "emails.magicSession.thanks": "תודה",
+    "emails.magicSession.signature": "צוות {{project}}",
+    "emails.recovery.subject": "איפוס סיסמא",
+    "emails.recovery.hello": "שלום {{name}}",
+    "emails.recovery.body": "עקוב אחר קישור זה כדי לאפס את סיסמת הפרויקט {{project}}.",
```
Comment (eldadfux) [inline app/config/locale/translations/he.json:20]:
```suggestion
    "emails.recovery.body": "עקוב אחר קישור זה כדי לאפס את סיסמתך ב-{{project}}.",
```

**Review by eldadfux — CHANGES_REQUESTED** [review]
Thanks for your PR! (and finally a translation I can read 😅)

I suggested a few fixes, please review them 🙏🏻

**Comment by Kokoden** [comment]
Always happy to see a fellow Hebrew speaker haha. I applied the changes you suggested, let me know if you need anything else fixed.

**Review by evinhaela — APPROVED** [review]

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to `eldadfux` on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags!

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) to help make Appwrite better.

**Pull request merged**